specification/Overview.md

```diff
@@ -89,54 +142,89 @@ organised by data category (saveframe).
       LYS HZ%, ...). Notably N-terminal -NH3+ would be 'H%', as would
       N-terminal -NH2.  '\*' would only be
       used where necessary, mainly for 'H\*', 'C\*', '\*',  ... (all protons, all
-      carbons,  all atoms, etc.).
+      carbons,  all atoms, etc.), or for atom names that do not use digits for
+      distinguishing stereochemistry (e.g. DNA/RNA H5' and H5'', where the
+      two-atom set is designated H5'\*).
 
-    10. IUPAC pseudoatom names (ALA MB, SER QB, etc.) are NOT expanded into
+    11. IUPAC pseudoatom names (ALA MB, SER QB, etc.) are NOT expanded into
     sets of atoms. They are reserved for the
     original meaning, i.e. a geometric point positioned at the centroid with zero
     van der Waals radius. A restraint to ALA MB would be different from one to
-    ALA HB%, and it would be an error to confuse them. E.g. ALA MB and ALA HB%
-    can both appear in the same shift list, if both are needed e.g. in different
-    restraint lists.
+    ALA HB%, and it would be an error to confuse them. Note that a restraint to
```

Comment:
I like to give some additional thoughts here. HB% and QB two different ways of looking at the same thing. What is experimentally observed is a nosey peak with degenerate/strongly overlapped chemical shift from a set of atoms and a restraints are constructed using this peak. How this degeneracy/overlap is treated(r-6 averaging or pseudo atom)  should be the choice of the software. I guess pseudo atom idea might have come from X-Ray where they have poor resolution(electron density) for a set of atoms.For  NMR I think its to better to stick to % to indicated the group. Its just a suggestion.